packages/fxa-settings/src/components/images/index.tsx

```diff
@@ -85,11 +87,35 @@ export const HeartsVerifiedImage = ({ className, ariaHidden }: ImageProps) => (
   />
 );
 
-export const RecoveryCodesImage = ({ className, ariaHidden }: ImageProps) => (
+export const BackupCodesImage = ({ className, ariaHidden }: ImageProps) => (
   <PreparedImage
     ariaLabel="Document that contains hidden text."
     ariaLabelFtlId="signin-recovery-code-image-description"
-    Image={RecoveryCodes}
+    Image={BackupCodes}
+    {...{ className, ariaHidden }}
+  />
+);
+
+export const BackupRecoveryPhoneImage = ({
+  className,
+  ariaHidden,
+}: ImageProps) => (
+  <PreparedImage
+    ariaLabel="Mobile device that receives a code by text message."
```

Comment:
I like these clear aria-labels! Agree we should move away from the "An illustration/image that..." format of the other labels.

Reply:
Agreed, especially since screen readers will read, "image".